routing.rst

```diff
@@ -531,6 +531,11 @@ and method: ``AppBundle\Controller\BlogController::showAction``. But if you
 follow some simple conventions, the logical name is more concise and allows
 more flexibility.
 
+.. tip::
+
+   For referring to the ``__invoke`` method of a controller, no need for
+   passing the method name, just use its fully-qualified namespace.
```

Comment:
I think we should reword this a bit:

> To refer to an action that is implemented as the `__invoke()` method of a controller class, you do not have to pass the method name, but can just use the fully qualified class name.

And we should then add an example for it imo.